deploy/docker/clickhouse-setup/docker-compose-local.yaml

```diff
@@ -12,11 +12,15 @@ services:
     container_name: query-service
     environment:
       - ClickHouseUrl=tcp://clickhouse:9000
+      - ALERTMANAGER_API_PREFIX=http://alertmanager:9093/api/
       - STORAGE=clickhouse
-      - SIGNOZ_LOCAL_DB_PATH=/root/signoz.db
+      - GODEBUG=netdns=go
+      - TELEMETRY_ENABLED=true
+      - SIGNOZ_LOCAL_DB_PATH=/var/lib/signoz/signoz.db
```

Comment:
@prashant-shahi what changed as we are moving the `SIGNOZ_LOCAL_DB_PATH` from `/root/signoz.db` to `/var/lib/signoz/signoz.db`?

Reply:
so it was not supposed to be working earlier?